crates/uv-configuration/src/target_triple.rs

```diff
@@ -570,6 +581,7 @@ impl TargetTriple {
             Self::Aarch64Manylinux238 => "",
             Self::Aarch64Manylinux239 => "",
             Self::Aarch64Manylinux240 => "",
+            Self::Wasm32Pyodide2024 => "#1",
```

Comment:
Is there a specific reason for this value?

Reply:
Not really but it's what Emscripten puts in this field. Added a comment and a link to Emscripten's libc.